server/test/repository/QuestionRepositoryTest.java

```diff
@@ -417,6 +418,15 @@ public void createOrUpdateDraft_managesPrimaryApplicantInfoTagsCorrectl()
         .isFalse();
   }
 
+  @Test
+  public void getQuestionsWithSimilarAdminNames_returnsSimilarAdminNames() {
+    resourceCreator.insertQuestion("name-question");
+    resourceCreator.insertQuestion("name-question-1");
+    resourceCreator.insertQuestion("name-question-2");
+    ImmutableList<String> adminNames = repo.getSimilarAdminNames("name-question");
```

Comment:
**suggestion:** Related to comment above, can we add a test that calls getSimilarAdminNames with "name-question-1"